README.md

```diff
@@ -125,10 +125,10 @@ fn main() -> io::Result<()> {
 ## Relationship to Tokio
 
 Romio is a fork of another Rust project called [Tokio][tokio]. The Tokio
-project uses an older version of the futures API which is not compatible with
+project would use an older version of the futures API which was not compatible with
 the new "async/await" syntax. In order to enable people to experiment with
-"async/await," Romio ports parts of the tokio project to the newer futures API
-which is compatible with that syntax.
+"async/await," Romio ported parts of the tokio project to the newer futures API
+which was compatible with that syntax.
```

Comment:
```suggestion
which is compatible with that syntax.
```
This is still current.

Reply:
Sequence of tenses?

Reply:
I'm not sure what that means, but feel free to disregard this suggestion; I don't feel strongly about it.